regimes/my/invoices.go

```diff
@@ -0,0 +1,53 @@
+package my
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+	"github.com/invopop/validation"
+)
+
+func validateInvoice(inv *bill.Invoice) error {
+	return validation.ValidateStruct(inv,
+		validation.Field(&inv.Supplier,
+			validation.By(validateSupplier),
+			validation.Skip,
+		),
+		validation.Field(&inv.Customer,
+			validation.By(validateCustomer),
+			validation.Skip,
+		),
+		validation.Field(&inv.Lines,
+			validation.Required,
+			validation.Length(1, 0),
+		),
+	)
+}
+
+func validateSupplier(val any) error {
+	obj, _ := val.(*org.Party)
+	if obj == nil {
+		return nil
+	}
+	return validation.ValidateStruct(obj,
+		validation.Field(&obj.TaxID,
+			validation.Required,
+			tax.RequireIdentityCode,
+			validation.Skip,
+		),
+	)
+}
+
+func validateCustomer(val any) error {
+	obj, _ := val.(*org.Party)
+	if obj == nil {
+		return nil
+	}
+	return validation.ValidateStruct(obj,
+		validation.Field(&obj.TaxID,
+			validation.Required,
+			tax.RequireIdentityCode,
```

Comment:
Are you sure this is a requirement? How are simplified invoices issued in Malasia when there is no customer?